superset/db_engine_specs/base.py

```diff
@@ -443,6 +452,9 @@ def get_dbapi_mapped_exception(cls, exception: Exception) -> Exception:
         """
         new_exception = cls.get_dbapi_exception_mapping().get(type(exception))
         if not new_exception:
+            # We are interested in just BigQuery exceptions
+            if cls.engine == EngineType.BIGQUERY:
```

Comment:
a better check is if the class has the function `parse_error_exception`

Reply:
They all have it because the base class have it, I had it that way initially but our hooks and linting rules don't like it because `cls` would be the base class and it checks with it instead of the BigQuery one for example. So, Was that or ignoring the rule if possible.